client/client-multi/client-c/src/kaa/kaa_context.h

```diff
@@ -95,7 +93,7 @@ typedef struct kaa_status_holder_t                  kaa_status_holder_t;
  * independently to perform API calls to specific subsystems.
  */
 typedef struct kaa_context_s {
-    kaa_status_holder_t         *status;                 /**< See @link kaa_status.h @endlink. */
+    kaa_status_t                *status;                 /**< See @link kaa_status.h @endlink. */
```

Comment:
kaa_status_holder_t is used to hide kaa_status_t  context from a user of Kaa library.
Why did you remove this holder?